tests/unit/test_hub.py

```diff
@@ -33,4 +33,8 @@ def test_hub_build_no_pymodules():
 
     args = set_hub_build_parser().parse_args(
         [os.path.join(cur_dir, 'hub-mwu-bad', 'fail-to-start'), '--pull', '--push', '--test-uses'])
-    assert not HubIO(args).build()['is_build_success']
\ No newline at end of file
+    assert not HubIO(args).build()['is_build_success']
+
+def test_hub_new():
+    args = set_hub_new_parser().parse_args('--type=\'app\'')
+    HubIO(args).new()
```

Comment:
is there anything we can assert to make sure it worked properly?

Reply:
This creates a template structure using the cookiecutter library function, only point of failure is if cookiecutter is not installed, in which case it raises exception, since this is just a very basic skeleton

Reply:
@rutujasurve94 please refer other places where I use `parse_args`, you basically give a list of string, in this case `['--type', 'app']`

Reply:
done